src/main/java/frc/trigon/robot/subsystems/shooter/Shooter.java

```diff
@@ -0,0 +1,71 @@
+package frc.trigon.robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.trigon.robot.utilities.Conversions;
+
+public class Shooter extends SubsystemBase {
+
+    // With eager singleton initialization, any static variables/fields used in the 
+    // constructor must appear before the "INSTANCE" variable so that they are initialized 
+    // before the constructor is called when the "INSTANCE" variable initializes.
+    WPI_TalonFX masterMotor = ShooterConstants.masterMotor;
+    WPI_TalonFX motor2 = ShooterConstants.motor2;
+    private int ballCount = 0;
+    public void setTargetVelocity(double velocity)
```

Comment:
change name to `setVelocity`

Reply:
well..... nothing is impossible... however practically you are right however the meaning of  `setVelocity` is still clear and is a better name

Reply:
I disagree as you're just setting the setpoint.

Reply:
last year I think we did `setVelocity` but I see what you are saying... if you want to leave it then its fine...